requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 pip~=22.0
-git+https://github.com/PennyLaneAI/pennylane.git@master
+git+https://github.com/PennyLaneAI/pennylane.git@lightning-op-math
```

Comment:
Leaving this here for later. Should be done AFTER two approvals
```suggestion
git+https://github.com/PennyLaneAI/pennylane.git@master
```

Reply:
We need the PennyLane PR to be merged first, right? What is the status of that?

Reply:
Not merged yet. I'm not sure if it will get merged today or not.